config/highlights.py

```diff
@@ -0,0 +1,16 @@
+"""
+Profile specific manually curated highlights.
+"""
+
+highlights = {
+    1: "reflects the highest proportion of the population (25% of households). This is also the oldest group with 40% of households having all occupants aged 65+, and a great percentage not on work. Across other characteristics, households in profile 1 are average.",
+    2: "has the annual lowest electricity and gas consumption, and has the greatest number of social housing tenants. This profile sees the gretest proportion of single occupancy households, and the least with children.",
+    3: "has second annual lowest electricity and gas usage, and the highest proportion of households where everyone is at work or studying.",
+    4: "has the second highest annual gas consumption. 36% are 65+ only households and 50% live in detached propertiles, with a higher than average proportion of properties built before 1930.",
+    5: "consists of a higher than average proportion of households with adults and children (at least 38%), and over 50% of households with 3 or more occupants. 6% have air conditioning units. 27% have smart heating controls.",
+    6: "this profile is mostly average across all characteristics.",
+    7: "Most households in profile 7 don’t consume gas. hile most electricity consumption profiles show a morning increase and an evening peak, profiles 10 exhibits particularly distinct trimodal peaks throughout the day. This profile has the highest proportion of renters (at least 30%) and highest proportion of flats (at leat 40%). At least 11% have solar panels, 8% have battery storage and 6% have heat pumps.",
```

Comment:
```suggestion
    7: "Most households in profile 7 don’t consume gas. While most electricity consumption profiles show a morning increase and an evening peak, profile 10 exhibits particularly distinct trimodal peaks throughout the day. This profile has the highest proportion of renters (at least 30%) and highest proportion of households living in flats (at leat 40%). At least 11% have solar panels, 8% have battery storage and 6% have heat pumps.",
```
You've mentioned profile 10 here - so I'm not sure if this is the correct numbers for profile 7 but I've edited the text either way.